index.md

```diff
@@ -5,46 +5,53 @@
 # Look in _data/ to modify actual data.
 layout: home
 ---
-## Welcome to the Hands-On UNIX SysAdmin DeCal, Spring 2018 Edition!
+## Welcome to the Linux System Administration DeCal, Fall 2018 Edition!
 This course will cover the basics of setting up and administering a
 production-quality Linux-based server environment in the manner of the EECS
 instructional machines (hivexx.cs.berkeley.edu) and those of the Open Computing
 Facility (tsunami.ocf.b.e). By the end of this course, we expect you to:
```

Comment:
Don't use the "b.e" shorthand here since it's unfamiliar to those outside the OCF

Reply:
Agreed